### DISCUSSION TITLE
select() function adding un-desired label text

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1351] [Q&A] [answered]
I'm trying to use the Wheels "select" function. It appears that it is inserting the property attribute into the label, prepending it to the select tag rendered. I already have a label for the select called "Entry Type". It appears to be trying to provide that label text for me and I should remove my manually added label, BUT... I don't want a label with underscore char in it. So I added a label attribute with a value "Entry Type" and that displayed without underscore char, BUT... it's not in my td tag for styling and alignment. I was going to add a class style to the td but Wheels is putting it in without a td (obviously it doesn't know I want it in a td tag.)

This image pretty much sums up my issue here:
![image](https://github.com/cfwheels/cfwheels/assets/4595721/eaaa6b27-3314-4f27-9fb6-086b9357c13e)

**Comment by neokoenig** [comment, marked as answer]
Think you can just do `label=false` on the `select()` ?

**Reply by cfcoder** [reply]
Thanks!
